patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/README.md

```diff
@@ -0,0 +1,65 @@
+# Dedicated Server Utils
+This directory contains utilities for managing a dedicated ***Linux*** server.
+
+## Docker vs Management Script
+Both the Docker container and the management script can install and update tModLoader and any mods. Docker is a service used to containerize, or isolate, a specific application from a host system. This is beneficial for both security and ease of use with more complicated services. If you would prefer more control over your server, then make use of `manage-tModLoaderServer.sh`.
+
+## Using The Script
+The `manage-tModLoaderServer.sh` script can be used to install tModLoader either directly from the GitHub release or from SteamCMD. The script is made to run fully standalone, so just download it to your server and run it. No other files from the repo are needed.
+
+### Installing tModLoader
+#### Via SteamCMD (recommended)
+* Ensure SteamCMD is installed and on your PATH. You can install SteamCMD from your package manager or [Valve's Wiki](https://developer.valvesoftware.com/wiki/SteamCMD).
+* Installation via SteamCMD can be performed using `./manage-tModLoaderServer.sh --install --username your_steam_username`.
+* You will be prompted for your password (and your 2fa code if applicable).
+* By default, tModLoader will be installed to `~/Steam/steamapps/common/tModLoader`. To specify an installation directory, use `--folder /full/path/from/root`
+
+#### From GitHub
+* Installation from the latest GitHub release can be performed using `./manage-tModLoaderServer.sh --install --github`.
```

Comment:
As above, just "Run" is fine. A bullet point below can say "The script will automatically use the latest Github release, which will be the same as the version released on Steam."